### DISCUSSION TITLE
Admin Panel?

### DISCUSSION THREAD

**Discussion by asili2001** [Tenkaklet/coderdojo#35] [Ideas]
Det skulle vara perfekt om vi har en admin panel där admin kan exempelvis skapa pages, kolla donation details, skicka/hantera emails. Uppdatera sidan och mycket mer. Så att man håller sig till en plattform. Undrar vad ni tycker?
I så fall behövs det att designa/utveckla frontend.

mvh
Ahmad Asili

**Comment by Tenkaklet** [comment]
Jag tycker att det är en bra idé men inte emails då det finns mallar och mycket redan på mailchimp. Men jag håller med om att man kan ha admin del för att ladda upp språk json filer och på det sättet.

**Comment by asili2001** [comment]
Vi kan också implementera mailchimp som kommer i sin tur används in i adminpanelen på coderdojo.

**Reply by Tenkaklet** [reply]
Jag håller med :)
